src/api/client/resolvers.ts

```diff
@@ -235,7 +239,7 @@ export const queryNodeStitchingResolvers = (
           info
         )
       } catch (error) {
-        ConsoleLogger.warn('Failed to resolve views field', { error })
+        SentryLogger.error('Failed to resolve views field', 'orionViewsResolver', error)
```

Comment:
```suggestion
        SentryLogger.error('Failed to resolve channel views', 'Channel.views resolver', error)
```

Reply:
I think you get the idea, we should do the same for all resolvers, not gonna add a comment for the rest